### PR TITLE
Add Theban jars code example

### PR DESCRIPTION
**Pull request by stefanv**
1/4 done

### CODE REVIEW

File: source/confidence_2.Rmd [hunk 1]
```diff
@@ -638,22 +638,32 @@ bootstrap procedure for estimating a confidence limit around the mean,
...
+data = np.array(
+    [11.4, 13.4, 13.5, 13.8, 13.9, 14.4, 14.5, 15.0, 15.1, 15.8, 16.0, 16.3,
+     16.5, 16.9, 17.0, 17.2, 17.5, 19.0]
+)
+
+k = 10000
+z = np.zeros(k)
+
+for i in range(k):
+    sample = rng.choice(data, 18)
```
Comment (matthew-brett) [inline source/confidence_2.Rmd:656]: I've gone for `size=18` from fairly early.

Comment (matthew-brett) [reply source/confidence_2.Rmd:656]: What do you think about:

```suggestion
    sample = rng.choice(data, size=18, replace=True)
    # In the line above, replace=True is the default, so we could leave it out to
    # get the same result.  We added it just to emphasize that bootstrap samples
    # are samples _with_ replacement.
```

?

**Review by matthew-brett — COMMENTED** [review]
Suggestions for code line.

**Comment by matthew-brett** [comment]
@stefanv  - where are we with this one?  Ready to merge?

**Comment by matthew-brett** [comment]
What do you think about merging this one in the meantime?

**Comment by stefanv** [comment]
Hi Matthew, please go ahead and commit any changes you'd like to see to this branch and merge. Thanks!

**Pull request merged**